build.gradle

```diff
@@ -46,7 +46,7 @@ dependencies {
     compileOnly("org.apache.commons:commons-text:1.10.0")
 
     // NBT-API
-    implementation("de.tr7zw:item-nbt-api:2.13.2") {
+    implementation("de.tr7zw:item-nbt-api:2.13.3-SNAPSHOT") {
```

Comment:
I'm going to wait until 2.13.3 is stable until I update skBee so we can also back this out for now.